docs/Getting.md

```diff
@@ -1,11 +1,23 @@
 **Cromwell Releases**
 
-The preferred way of getting Cromwell releases is at the [GitHub Releases](https://github.com/broadinstitute/cromwell/releases/latest) page. 
+Cromwell releases are available at the [GitHub Releases](https://github.com/broadinstitute/cromwell/releases/latest) page. 
 You are strongly encouraged to use the latest release of Cromwell whenever possible.
 
+Cromwell is distributed is a conda package on [conda-forge](https://conda-forge.org/).
```

Comment:
OOI do you know who maintains the conda package for Cromwell? As someone who doesn't use conda, this seems like magic to me (and so I'm a little nervous about putting it in docs!)

If it's not automatic, could we say "Cromwell is also published with community support as a conda package ..."?

Reply:
[The cromwell feedstock maintainers](https://github.com/conda-forge/cromwell-feedstock#feedstock-maintainers). That includes me.
Our job is very easy though. Every time there is a new Cromwell release, the conda-forge autotick bot will update the recipe and ping us. We just have to check the recipe and click the merge button. We have not had a not-working PR from the autotick bot yet. Since there are four of us, the conda package is usually published within a day. Plus the conda version comes with a great wrapper.
As long as the Cromwell team keeps the naming scheme of the JARs the same, it should be fully automatic. If you want, you could be added as recipe maintainer.